### PR TITLE
Add appendToConfig and example to mobile-config.md

### PR DESCRIPTION
**Pull request by pathway27**

### CODE REVIEW

**Comment by lorensr** [comment]
Thanks, looks good! Would you actually please switch it back to master? vNEXT is for unreleased API changes, and this was in `1.3.4.2`

https://github.com/meteor/meteor/blob/devel/History.md#v1342
https://github.com/meteor/meteor/pull/7307

File: source/api/mobile-config.md [hunk 1]
```diff
@@ -49,6 +49,15 @@ App.configurePlugin('com.phonegap.plugins.facebookconnect', {
...
 });
+
+// Add custom tags for a particular PhoneGap/Cordova plugin
+// to the end of generated config.xml.
+// Universal Links is shown as an example here.
+App.appendToConfig('
+  <universal-links>
+    <host name="localhost:3000" />
+  </universal-links>
+');
```
Comment (lorensr) [inline source/api/mobile-config.md]: These `'`s should be \`s

**Comment by pathway27** [comment]
Swapped to master and changed to \`s.  ^.^

**Comment by lorensr** [comment]
Thank you 😊

**Pull request merged**